/**
 * Tests for correct functionality when a {@link org.apache.oozie.action.ActionExecutorException.ErrorType#TRANSIENT}
 * error is generated when trying to start an action. </p> It first generates a {@link
 * org.apache.oozie.action.ActionExecutorException.ErrorType#TRANSIENT} error. 2 retries with an interval of 10
 * seconds between them are allowed. The state of the action is checked after each attempt to be at {@link
 * org.apache.oozie.WorkflowActionBean.Status#START_RETRY}. Error message and Error code for the action are
 * verified. </p> After the configured number of retry attempts, the job and actions status are checked to be {@link
 * org.apache.oozie.client.WorkflowJob.Status#SUSPENDED} and {@link org.apache.oozie.WorkflowActionBean.Status#END_MANUAL}
 * respectively. The error message and code are verified again.
 *
 * @throws Exception
 */
public void testStartTransient() throws Exception {
    _testTransient("start.transient", WorkflowActionBean.Status.START_RETRY, WorkflowActionBean.Status.START_MANUAL, "start");
    assertTrue(true);
}
